satpy/composites/__init__.py

```diff
@@ -1410,6 +1416,10 @@ def __init__(self, name, filename=None, area=None, **kwargs):
     def __call__(self, *args, **kwargs):
         """Call the compositor."""
         from satpy import Scene
+        # Check if filename exists, if not then try from SATPY_ANCPATH
+        if (not os.path.isfile(self.filename)):
+            self.filename = os.path.join(get_environ_ancpath(), self.filename)
+        print(self.filename)
```

Comment:
We should probably remove that print statement...

Reply:
Ahem. Just checking you're awake ;-)

Reply:
of course you are :)